add setting screen address logic

AHD-WAEL opened the pull request:
- add new address
- update existing address by user id and address id
- delete from address list by user id and address id
- get all addresses according to user id
- change address cell to nib file cell
- refactor settings ui constraints (back buttons, headers)

AHD-WAEL committed:
add setting screen address logic

- add new address
- update existing address by user id and address id
- delete from address list by user id and address id
- get all addresses according to user id
- change address cell to nib file cell
- refactor settings ui constraints (back buttons, headers)

alaagawish reviewed (changes requested):
warning message while editing address

alaagawish commented:
editing address by swiping

alaagawish commented:
check posting new address

AHD-WAEL committed:
refactoring what came from Pr

- check posting new address
- warning message while editing address

AHD-WAEL committed:
refactoring what came from Pr

- editing address by swiping
- change title in add address according to the screen came from

AHD-WAEL committed:
change alert msg in add address screen

alaagawish committed:
Merge branch 'develop' into user_address

alaagawish reviewed (changes requested):
move function from constants to utils

AHD-WAEL committed:
refactor json coding functions

AHD-WAEL committed:
Merge branch 'user_address' of https://github.com/alaagawish/InDoor-User into user_address

# Conflicts:
#	InDoor-User/Utils/Constants.swift
